### ISSUE TITLE
requests isn't a dependency

### ISSUE THREAD

**Issue by lsankar4033** [paradigmxyz/paradigm-data-portal#1]
probably need to add `requests` as a dep in `pyproject.toml`

**Comment by 0xboredagain** [comment]
+1 getting the same error. 
```
❯ pdp download ethereum_contracts
Downloading dataset: ethereum_contracts
No module named 'requests'
```

**Comment by sslivkoff** [comment]
ty fixed in 3abeb6b471423201f72d74cff4af355ecaf06fe8
